app/forms/consent_form.rb

```diff
@@ -10,13 +10,24 @@ class ConsentForm < QuestionsForm
     :birth_date_day,
     :primary_first_name,
     :primary_last_name,
-    :primary_last_four_ssn
+    :primary_ssn,
+    :primary_tin_type
   )
+  set_attributes_for :confirmation, :primary_ssn_confirmation
+  set_attributes_for :virtual, :ssn_no_employment
```

Comment:
What is this used for? Do we need to save it as a tin_type value before validating like in `app/forms/ctc/dependents/info_form.rb:37`

Reply:
And if we need to include it we should add the checkbox into the view and make the spouse_consent form + view consistent